scripts/Deploy.s.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+import "contracts/FlatDirectory.sol";
+import "contracts/FlatDirectoryFactory.sol";
+
+contract Deploy is Script {
```

Comment:
why do we need a deployment script?

Reply:
Provide users with an option to deploy the relevant contracts with a single click.